### DISCUSSION TITLE
how to self host?

### DISCUSSION THREAD

**Discussion by ShyMaster202** [yume-chan/ya-webadb#442] [Q&A] [answered]
hi, i would like to know how to self host this project as i'd also like to make my own adjustments over time(mainly style changes like adding a dark mode, but also adding some other functionality)
so could anyone give me general tips on hosting this?

**Comment by yume-chan** [comment, marked as answer]
See https://github.com/yume-chan/ya-webadb/blob/13cba06937cf4f70d704c867e5a873d200ccfae6/CONTRIBUTING.md
